cli/user/src/main/java/io/pravega/cli/user/UserCLIRunner.java

```diff
@@ -7,33 +7,43 @@
  *
  *     http://www.apache.org/licenses/LICENSE-2.0
  */
-package io.pravega.test.integration.demo.interactive;
+package io.pravega.cli.user;
 
 import ch.qos.logback.classic.Level;
 import ch.qos.logback.classic.LoggerContext;
+import com.google.common.annotations.VisibleForTesting;
 import com.google.common.base.Strings;
+import io.pravega.cli.user.config.ConfigCommand;
+import io.pravega.cli.user.config.InteractiveConfig;
+import lombok.Cleanup;
+import lombok.val;
+import org.slf4j.LoggerFactory;
+
 import java.util.Arrays;
 import java.util.Collection;
 import java.util.Collections;
 import java.util.Comparator;
 import java.util.Scanner;
 import java.util.stream.Collectors;
-import lombok.Cleanup;
-import lombok.val;
-import org.slf4j.LoggerFactory;
 
 /**
  * Interactive CLI Demo Tool for Pravega.
  */
-public class InteractiveDemo {
+public class UserCLIRunner {
     private static final String CMD_HELP = "help";
     private static final String CMD_EXIT = "exit";
 
-    public static void main(String[] args) throws Exception {
+    public static void main(String[] args) {
+        doMain(args);
+        System.exit(0);
+    }
+
+    @VisibleForTesting
+    public static void doMain(String[] args) {
         LoggerContext context = (LoggerContext) LoggerFactory.getILoggerFactory();
         context.getLoggerList().get(0).setLevel(Level.ERROR);
 
-        System.out.println("Pravega Interactive Demo Tool.");
+        System.out.println("Pravega User CLI Tool.");
         System.out.println("\tUsage instructions: https://github.com/pravega/pravega/wiki/Interactive-Demo-Tool\n");
```

Comment:
Let's fix this reference too once we rename that page.

Reply:
Done.